instrumentation/spring/spring-batch-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/batch/v3_0/basic/SpringBatchTest.java

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.basic;
+
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.satisfies;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.runner.JobRunner;
+import io.opentelemetry.sdk.testing.assertj.SpanDataAssert;
+import io.opentelemetry.sdk.testing.assertj.TraceAssert;
+import io.opentelemetry.sdk.trace.data.StatusData;
+import io.opentelemetry.semconv.ExceptionAttributes;
+import java.util.Collections;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.batch.core.JobParameter;
+
+abstract class SpringBatchTest {
+
+  private final JobRunner runner;
+
+  @RegisterExtension
+  static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  public SpringBatchTest(JobRunner runner) {
+    this.runner = runner;
+  }
+
+  @Test
+  void should_trace_tasklet_job_step() {
```

Comment:
Why don't use lower camel case here?

Reply:
was just so much harder to read compared to the groovy version - I know it's not the convention, but this method isn't called anywhere

Reply:
related https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11999#discussion_r1721966755

@zeitlinger may be better to follow existing conventions during refactoring and propose changes to existing conventions as a separate issue